### DISCUSSION TITLE
Update to Chinese (Traditional) locale [zh_TW]

### DISCUSSION THREAD

**Discussion by SeasonChannel32** [aclap-dev/video-downloadhelper#556] [Language & Translation]
Translation for Video DownloadHelper Version 9
[zh_TW.txt](https://github.com/aclap-dev/video-downloadhelper/files/15412263/zh_TW.txt)

**Comment by z17898** [comment]
[zh_TW.txt](https://github.com/aclap-dev/video-downloadhelper/files/15450025/zh_TW.txt)
I edit some sentences
I think It would be clearer and easier to understand

**Reply by paulrouget** [reply]
Thank you!

**Comment by SeasonChannel32** [comment]
Translation for Video DownloadHelper Version 9.0.2.3
have edit to be clearer some sentences
[zh_TW.txt](https://github.com/aclap-dev/video-downloadhelper/files/15482080/zh_TW.txt)

**Comment by paulrouget** [comment]
Thank you. this will be in v9.1.
